Classes/Issues/Merge/MergeButton.swift

```diff
@@ -94,6 +98,10 @@ final class MergeButton: UIView {
         } else {
             activityView.stopAnimating()
         }
+
+        accessibilityLabel = title
+        // FIXME: This is currently not accessible in VoiceOver.
+        optionIconView.accessibilityLabel = NSLocalizedString("More merging options", comment: "More options button for merging")
```

Comment:
I think we can merge this even though this is not accessible after this patch, as it isn't accessible at the moment either. The problem is that this `subView` seems not to be separately accessible from the `superview`, even though it's z-axis is higher.

If anyone has any pointers to fix this, let me know!